products/spanner/api.yaml

```diff
@@ -52,25 +52,36 @@ objects:
     description: |
       An isolated set of Cloud Spanner resources on which databases can be
       hosted.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/spanner/'
+      api: 'https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances'
     exports:
       - name
     transport: !ruby/object:Api::Resource::Transport
       decoder: decode_response
     collection_url_response: !ruby/object:Api::Resource::ResponseList
       items: 'instances'
+<%= indent(compile_file({}, 'templates/global_async.yaml.erb'), 4) %>
     properties:
       - !ruby/object:Api::Type::String
         name: 'name'
```

Comment:
I think this needs to be marked `input`, and `required`? From the website docs:

> Required. A unique identifier for the instance, which cannot be changed after the instance is created. Values are of the form projects/<project>/instances/[a-z][-a-z0-9]*[a-z0-9]. The final segment of the name must be between 6 and 30 characters in length.

For Terraform, we can preserve the generated name w/ overrides of course!